packages/orbit-components/src/Popover/Popover.stories.tsx

```diff
@@ -89,7 +86,7 @@ type Story = StoryObj<typeof Popover>;
 export const Default: Story = {
   render: args => (
     <Popover {...args} content={content}>
-      <Button type="secondary" iconRight={<ChevronDown />}>
+      <Button asComponent="div" type="secondary" iconRight={<ChevronDown />}>
```

Comment:
When polishing a11y in SB, maybe I would also add attributes to the `Stepper` component (line `40`, `47`) to fix violations when popover is opened. wdyt?
something like this should be enough, I guess:
```
    ariaLabelValue="Number of adult passengers"
    titleIncrement="Add adult passenger"
    titleDecrement="Remove adult passenger"
```
and
```
    ariaLabelValue="Number of children passengers"
    titleIncrement="Add children passenger"
    titleDecrement="Remove children passenger"
```

Reply:
That's a bit out of scope… but ok, I'll do it